src/docs/product/releases/health/setup.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Set Up
+sidebar_order: 40
+description: "Learn about setting up your SDK to enable Release Health."
+---
+
+To enable reporting of release health, use the link below to quickly configure your SDK.
+
+## Supported SDKs
```

Comment:
Rust and Python should be here too

Reply:
+ JavaScript https://docs.sentry.io/platforms/javascript/configuration/releases/#release-health

Reply:
I will add them, and change the Release Health information for these SDKs in the common content to `supported`